P/PDAL/bundled/patches/mingw-portableendian.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/pdal/util/portable_endian.hpp b/pdal/util/portable_endian.hpp
+index b0e753f32..e9ea563b7 100644
+--- a/pdal/util/portable_endian.hpp
++++ b/pdal/util/portable_endian.hpp
+@@ -54,6 +54,9 @@
+ #   include <winsock2.h>
+                      
+ #   if BYTE_ORDER == LITTLE_ENDIAN
++
++#       define htonll(x) ((1==htonl(1)) ? (x) : ((uint64_t)htonl((x) & 0xFFFFFFFF) << 32) | htonl((x) >> 32))
```

Comment:
This patch is the from the set I pulled from the other user branch. This is slightly different from the other version but same idea. Which one should we use?

Reply:
I don't know, I don't have any preferences, as long as they all give the right results :smile:

Reply:
I went with the one from the the `whereskenneth` patchset since I know that one builds :laughing: